tests/test_on_data/ophys/test_imaging_interfaces.py

```diff
@@ -51,167 +51,177 @@ class TestTiffImagingInterface(ImagingExtractorInterfaceTestMixin):
-                channel_name="Channel 4",
-            ),
-            "photon_series_name": "TwoPhotonSeriesChannel4",
-            "imaging_plane_name": "ImagingPlaneChannel4",
-        },
-    ],
-)
-class TestScanImageImagingInterfaceMultiPlaneCase(ScanImageMultiPlaneImagingInterfaceMixin):
+class TestScanImageImagingInterfaceMultiPlaneChannel1(ScanImageMultiPlaneImagingInterfaceMixin):
     data_interface_cls = ScanImageImagingInterface
     interface_kwargs = dict(
-        file_path=str(OPHYS_DATA_PATH / "imaging_datasets" / "ScanImage" / "scanimage_20220923_roi.tif"),
+        file_paths=[OPHYS_DATA_PATH / "imaging_datasets" / "ScanImage" / "scanimage_20220923_roi.tif"],
         channel_name="Channel 1",
+        interleave_slice_samples=True,
     )
     save_directory = OUTPUT_PATH
 
     photon_series_name = "TwoPhotonSeriesChannel1"
     imaging_plane_name = "ImagingPlaneChannel1"
     expected_two_photon_series_data_shape = (6, 256, 528, 2)
-    expected_rate = 29.1248
-    expected_starting_time = 0.0
+    expected_rate = None  # This is interleaved data so the timestamps are written
+    expected_starting_time = None
 
     def check_extracted_metadata(self, metadata: dict):
         assert metadata["NWBFile"]["session_start_time"] == datetime(2023, 9, 22, 12, 51, 34, 124000)
 
 
-@parameterized_class(
-    [
-        {
-            "interface_kwargs": dict(
-                file_path=str(OPHYS_DATA_PATH / "imaging_datasets" / "ScanImage" / "scanimage_20220923_roi.tif"),
-                channel_name="Channel 1",
-                plane_name="0",
+class TestScanImageImagingInterfaceMultiPlaneChannel4(ScanImageMultiPlaneImagingInterfaceMixin):
+    data_interface_cls = ScanImageImagingInterface
+    interface_kwargs = dict(
+        file_paths=[OPHYS_DATA_PATH / "imaging_datasets" / "ScanImage" / "scanimage_20220923_roi.tif"],
+        channel_name="Channel 4",
+        interleave_slice_samples=True,
+    )
+    save_directory = OUTPUT_PATH
+
+    photon_series_name = "TwoPhotonSeriesChannel4"
+    imaging_plane_name = "ImagingPlaneChannel4"
+    expected_two_photon_series_data_shape = (6, 256, 528, 2)
+    expected_rate = None  # This is interleaved data so the timestamps are written
+    expected_starting_time = None
+
+    def check_extracted_metadata(self, metadata: dict):
+        assert metadata["NWBFile"]["session_start_time"] == datetime(2023, 9, 22, 12, 51, 34, 124000)
+
+
+class TestScanImageImagingInterfaceSinglePlaneCase(ScanImageSinglePlaneImagingInterfaceMixin):
+    data_interface_cls = ScanImageImagingInterface
+    save_directory = OUTPUT_PATH
+    expected_two_photon_series_data_shape = (6, 256, 528)
+
+    @pytest.fixture(
+        params=[
+            dict(
+                interface_kwargs=dict(
+                    file_path=str(OPHYS_DATA_PATH / "imaging_datasets" / "ScanImage" / "scanimage_20220923_roi.tif"),
+                    channel_name="Channel 1",
+                    plane_name="0",
```

Comment:
I thought `plane_name` is deprecated?

Reply:
Thanks, I will change this to use the new one.

Reply:
Done.